src/components/FBMessage/FBMessage.module.css

```diff
@@ -0,0 +1,9 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
```

Comment:
Is it nessessory to include a module for CSS even if there is only one HTML tag? I don't really follow. Isn't it easier just to add this line of code (tailwind css) to div inside the very component?  I think you are doing extra work for no profit whatsoever. Splitting the logic doesn't really needed here because it doesn't reduce readability of the code but it really does reduce folders readability. It is ok if you are more familiar with this approach and can do that if you want, but the main purpose of tailwindCSS is removing the need in extra files.

Reply:
#### TLDR
As I know it is a common practice to make a CSS module corresponding to every React Component. I just follow it

#### Q&A

> Is it nessessory to include a module for CSS even if there is only one HTML tag?

No

> Isn't it easier just to add this line of code (tailwind css) to div inside the very component?

Yes, for the `FBComponent`. Because there is only one div element with corresponding classname

#### Recap

I think to prefer a lack of CSS module for `FBComponent` is a matter of taste
Anyway it is easy for me to remove it. You decide